docker/docker-compose.yml

```diff
@@ -176,13 +176,8 @@ services:
   # Kafka is used for streaming functionality.
   # ZooKeeper is required by Kafka and Pinot
   kafka-zookeeper:
-    image: hypertrace/kafka-zookeeper:0.1.4
+    image: hypertrace/kafka-zookeeper
```

Comment:
We definitively need the CI running nightly on this.

Reply:
dude it is awesome you volunteer!

Reply:
seriously though I think we will change this to main tag once that is published on commit.. probably we need matrix latest and main to know impact

Reply:
ps not really sure why all but 2 are latest. but I will update those once touch them